packages/expo-cli/src/commands/build/logBuildMigration.ts

```diff
@@ -0,0 +1,50 @@
+import chalk from 'chalk';
+
+import Log from '../../log';
+import * as TerminalLink from '../utils/TerminalLink';
+
+const daysUntilDate = (date: string) => {
+  const today = new Date();
+  const eventDate = new Date(date);
+  const timeDiff = eventDate.getTime() - today.getTime();
+  if (timeDiff < 0) {
+    return 0;
+  }
+  const diffDays = Math.ceil(timeDiff / (1000 * 3600 * 24));
+  return diffDays;
+};
+
+export function logBuildMigration(platform: string) {
+  Log.newLine();
+  const command = chalk.bold(`expo build:${platform}`);
+  Log.log(
+    `${command} has been replaced by ${chalk.bold`eas build`}. ${chalk.dim(
+      TerminalLink.learnMore(`https://blog.expo.dev/turtle-goes-out-to-sea-d334db2a6b60`)
+    )}`
+  );
+  Log.newLine();
+  Log.log('Run the following:');
```

Comment:
these instructions look really nice but in practice they won't work for a lot of projects. we should guide them to https://docs.expo.dev/build-reference/migrating/ instead